app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.tsx

```diff
@@ -217,14 +219,14 @@ const TableVisualizationRow: React.FC<{
     const userCountsList =
       branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE]["all"];
     const metricDataList = metricData[branchComparison]["all"];
-    const significance = metricData["significance"];
 
     userCountsList.sort(formattedAnalysisPointComparator);
     metricDataList.sort(formattedAnalysisPointComparator);
 
     metricDataList.forEach((dataPoint: FormattedAnalysisPoint, i: number) => {
       const { lower, upper, point, count } = dataPoint;
       const userCountMetric = userCountsList[i]["point"];
+      const significance = metricData["significance"]?.[window][i + 1];
```

Comment:
Should we update any Storybook states in `TableVisualizationRow` to show what `overall` vs `weekly` looks like?

Reply:
the items in `TableVisualizationRow` don't look different if it's `weekly` vs. `overall`. But the difference can be seen in the storybook of `TableResultsWeekly`